.circleci/config.yml

```diff
@@ -321,7 +321,7 @@ orbs:
                 if << parameters.unit_test >> ; then
                   docker run -d --env AWS_ACCESS_KEY_ID='-' --env AWS_SECRET_ACCESS_KEY='-' --name tests << parameters.ecr_url >>/<< parameters.ecr_repository_name_prefix >>_app:latest
                   docker exec tests docker-php-ext-enable xdebug
-                  docker exec tests /app/vendor/bin/phpunit
+                  docker exec tests /app/vendor/bin/phpunit -d memory_limit=256M
```

Comment:
Hey - do we want the memory limit change for the phpunit tests in this pr?

Reply:
i needed it in to test the build through, can remove this after.

Reply:
👍